testsuites/lucene/src/main/resources/org/eclipse/rdf4j/sail/yeastract_raw.ttl

```diff
@@ -0,0 +1,1436 @@
+
+<urn:raw:yeastract#Yeast_id> <http://www.w3.org/2000/01/rdf-schema#label> "yeast_id" ;
```

Comment:
Where does this test data come from? If it's third-party we'll need to know its license, and log a CQ for its inclusion.